charts/pega/README.md

```diff
@@ -244,6 +244,19 @@ A `volumeClaimTemplate` may be configured for any tier to allow for persistent s
 
 The `deploymentStrategy` can be used to optionally configure the [strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) for any tiers deployed as a Kubernetes Deployment. This value will cannot be applied to StatefulSet deployed tiers which use the `volumeClaimTemplate` parameter.
 
+### Environment variables
+
+While Pega supports a variety of configuration options for cluster wide and application settings, in some cases you may want to pass a specific environment variable into your deployment on a tier-by-tier basis.  This can be accomplished by specifying a custom `env` block for your tier as shown in the example below.
```

Comment:
Pega supports a variety of configuration options for cluster-wide and application settings. In cases when you want to pass a specific environment variable into your deployment on a tier-by-tier basis, you specify a custom `env` block for your tier as shown in the example below.

Reply:
Updated